forte/data/ontology/ontology_code_generator.py

```diff
@@ -454,38 +461,82 @@ def parse_ontology_spec(self, json_file_path: str,
         # Extract imported json files and generate ontology for them.
         json_imports: List[str] = spec_dict.get("import_paths", [])
 
-        for import_file in json_imports:
-            import_json_file = utils.search_in_dirs(import_file,
+        # Get location of installed forte
```

Comment:
Line 464 to 494 are used to resolve imports, consider have a new function for this?

Reply:
Modularized this otherwise unwieldy part of the code.